release-notes/4.7.24.md

```diff
@@ -0,0 +1,203 @@
+# CiviCRM 4.7.24
+
+Released September 6, 2017;
+
+- **[Synopsis](#synopsis)**
+- **[Features](#features)**
+- **[Bugs resolved](#bugs)**
+- **[Miscellany](#misc)**
+- **[Credits](#credits)**
+- **[Feedback](#feedback)**
+
+## <a name="synopsis"></a>Synopsis
+
+| *Does this version...?*                                     |         |
+|:----------------------------------------------------------- |:-------:|
+| Fix security vulnerabilities?                               |         |
+| Change the database schema?                                 |         |
+| Alter the API?                                              |         |
+| Require attention to configuration options?                 |         |
+| Fix problems installing or upgrading to a previous version? |         |
+| Introduce features?                                         |         |
+| Fix bugs?                                                   |         |
+
```

Comment:
@agh1 it might be good to mention up here that we have Experimental php7.1 support from this release, may also want to mention the setting of sunset date of php5.3 and php5.4